utils/gyb_syntax_support/Token.py

```diff
@@ -89,39 +93,46 @@ def __init__(self, name, text):
     Keyword('__DSO_HANDLE__', '__DSO_HANDLE__'),
     Keyword('Wildcard', '_'),
     Token('PoundAvailable', 'pound_available', text='#available',
-          is_keyword=True),
+          is_keyword=True, classification='Keyword'),
```

Comment:
Can we share the token classification information with the C++ side as well? We should specify them only once in here.

Reply:
OK. Never mind then.